include/openthread/tcat.h

```diff
@@ -67,6 +67,11 @@ extern "C" {
 #define OT_TCAT_MAX_SERVICE_NAME_LENGTH \
     15 ///< Maximum string length of a UDP or TCP service name (does not include null char).
 
+#define OT_TCAT_ADVERTISEMENT_MAX_LEN 29 ///< Maximum length of TCAT advertisement.
+#define OT_TCAT_VERSION 0x5              ///< TCAT version number.
```

Comment:
This definition is not needed - TCAT uses the Thread Version number directly in the first nibble.This is `OPENTHREAD_CONFIG_THREAD_VERSION` .

Reply:
Fixed.